recipes/magics-metview/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "Magics" %}
+{% set version = "4.2.0" %}
+
+package:
+  name: magics-metview
+  version: {{ version }}
+
+source:
+  url: https://confluence.ecmwf.int/download/attachments/3473464/{{ name }}-{{ version }}-Source.tar.gz
+  sha256: d45701515d3329e1b3944d535c4d19f834d22a747a30da1d8c88b80affeb70cd
+  patches:
+    - 0001-Find-proj_6_0.lib-if-proj.lib-isn-t-found.patch
+    - 0002-Fix-for-Windows-build.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
```

Better get make from conda-forge rather than using the host:
```suggestion
    - cmake
    - make
```
(For example, the container for POWER builds doesn't have it included)